README.md

```diff
@@ -93,3 +93,89 @@ Grafana uses authentication, which, for this workshop, is configured in the `doc
 username: ecosia
 password: workshop
 ```
+
+### Bonus Material: Data Types in Prometheus
```

Comment:
Maybe this should be Histograms in Prometheus rather than Data Types?

Reply:
good point!